SampleApp/SampleApp.iOS/SampleApp.iOS.csproj

```diff
@@ -86,11 +86,8 @@
   <ItemGroup>
     <PackageReference Include="Xamarin.Forms" Version="4.5.0.657" />
     <PackageReference Include="Xamarin.Essentials" Version="1.5.3.1" />
-    <PackageReference Include="urbanairship.ios.core">
-      <Version>13.5.2</Version>
-    </PackageReference>
-    <PackageReference Include="urbanairship.ios.messagecenter">
-      <Version>13.5.2</Version>
+    <PackageReference Include="urbanairship.ios">
```

Comment:
whats the dealio above?

Reply:
Oh, I should have mentioned this in the description. one of the problems I noticed with the sample app was that it didn't include automation to begin with, so if someone was copying its structure they might end up with an app that won't be able to get IAA. Since we don't provide direct IAA APIs in the .NETStandard library I figured the best thing would be to just use the umbrella package.

Reply:
but yeah the version is wrong right now since it came from the local build